### DISCUSSION TITLE
[Bug]: slow performance on Safari Browser since 5.8

### DISCUSSION THREAD

**Discussion by nafets227** [mbentley/docker-omada-controller#354] [Help] [answered]
### Controller Version

v5.8 up to v5.12

### Describe the Bug

UI is unusable slow on Safari browser, page load times ~5s.

Nothing the container can solve, maybe good to mention it in known issues.
See also https://community.tp-link.com/en/business/forum/topic/245226

### Expected Behavior

Page load times < 1 s, 

### Steps to Reproduce

1. Install v5.8+
2. Open the URL in Safari
3. Each click takes ~5s to respond

### How You're Launching the Container

```plain
in Kubernetes
```

### Container Logs

```plain
not relevant
```

### MongoDB Logs

```plain
not relevant
```

### Additional Context

As said, not a container issue just worth to mention it in known issues.

**Comment by mbentley** [comment]
Yeah, Safari performance looks pretty terrible compared to Firefox or Chrome.  Have you created an issue on the TP-Link forums or found one with a similar issue that I could reference in a known issue?

**Reply by nafets227** [reply]
yes, look at https://community.tp-link.com/en/business/forum/topic/245226 already linked in the description.

5.7.x works fine, 5.8 until at least 5.12 not.

**Reply by mbentley** [reply]
I see no references to Safari slowness in that link.

**Reply by mbentley** [reply, marked as answer]
https://community.tp-link.com/en/business/forum/topic/619304?replyId=1255404 does.

**Reply by nafets227** [reply]
my apologies, I must have copied the wrong link.
